Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace BootCamp.Chapter
+{
+    internal struct Credentials
+    {
+        public readonly string Username;
+        public readonly string Password;
+
+        public Credentials(string username, string password)
+        {
+            Username = username;
```

Comment:
It would be nice to validate against null or empty

Reply:
As requested, made change to validate username and password against null or empty.